db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
```

Comment:
**PLEASE DO NOT PUSH SCHEMA.rb** Before raising please you review once

Reply:
?

Reply:
I have added this file to git ignore file but I don't know, how it was pushing into the repo.
next time I will take care of it.